openslides/agenda/models.py

```diff
@@ -163,7 +163,7 @@ def get_title(self):
         Return the title of this item.
         """
         if not self.content_object:
-            agenda_title = self.title
+            agenda_title = self.title or ""
```

Comment:
If the title is None the PDF should print the title of the related item. This looks like an error in the construction of the elements of this app.